### ISSUE TITLE
Data column(s) for axis #0 cannot be of type string

### ISSUE THREAD

**Issue by jhshose** [google/google-visualization-issues#2867]
```
<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
<%@ taglib prefix="ui" uri="http://egovframework.gov/ctl/ui"%>
<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<title>URL Monitor</title>
<!-- jQuery -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>

<!-- jQuery UI -->
<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
<link rel="shortcut icon" href="#">

</head>

<body class='chart'>
	<input type="hidden" id="urlNo" name="urlNo" value="${infoLogVO.urlNo}"/>

	<!-- 차트 표시될 영역 -->
	<div class="panel">
		<div class="head">
			<strong>상태 및 응답 시간 현황</strong>
		</div>
		<div class="body">
			<c:if test="${total == 0 }"></c:if>
			<c:if test="${total > 0 }">
				<div id="result"></div>
			</c:if>

		</div>
	`</div>`
</body>
</html>

<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
<script type="text/javascript">
google.charts.load('current', {packages: ['corechart'], 'callback': drawVisualization});
google.charts.setOnLoadCallback(drawVisualization);

function drawVisualization(){
	var no = $("#urlNo").val();

	$.ajax({
		url: "/googleLineChart.do",
		type: "GET",
		data: {
			urlNo : no
		},
		dataType: "json",
		success: function(json){

			//console.log(json.data);
			var timeArray = [];
			var dateTimeArray = [];
			var dateArray = [];

			$.each(json.data, function(i, obj){
				dateTimeArray[i] = obj.datetime;
				timeArray[i] = obj.time;
				dateArray[i] = obj.date;
			})

			var data = google.visualization.arrayToDataTable([
            	[dateTimeArray, timeArray, dateArray]
            ]);

			var view = new google.visualization.DataView(data);
			view.setColumns([0,{
				calc : "stringify",
				sourceColumn : 1,
				type : "String",
				role : "annotation"
				},1,2 ]);

			var options = {
				title : '라인차트',
				hAxis : {title : 'time'}, //y축
				vAxis : {title : 'datetime'}, //x축
				//seriesType : 'bars',
				//series: {5: {type: 'line'}}
			}

			var chart = new google.visualization.LineChart(document.getElementById('result'));

			chart.draw(view, options);
		}
	})
}

</script>
```

구글 차트 api 적용해서 차트 개발중입니다.

에러내용은 이러하면서 차트가 뜨지를 않습니다.
Data column(s) for axis #0 cannot be of type string

ㅜㅜ

![구글 콘솔](https://user-images.githubusercontent.com/38000701/96827514-b75ab580-1470-11eb-8ee1-1f8534e8ba92.PNG)
